queenbee/cli/repository/__init__.py

```diff
@@ -36,3 +38,5 @@ def main(ctx):
     pass
 
 main.add_command(index)
+main.add_command(serve)
+main.add_command(initialise)
```

Comment:
possible to change this to `init` so we don't have American vs British spelling discussions! :)

Reply:
Yeah for sure. Feel free to push a PR with changes. Judging by the widespread spelling mistakes in my code I'm in not position to push back on this :upside_down_face: